cl/alerts/management/commands/cl_send_alerts.py

```diff
@@ -63,10 +65,32 @@ def send_alert(user_profile, hits):
     txt_template = loader.get_template("alert_email.txt")
     html_template = loader.get_template("alert_email.html")
     context = {"hits": hits}
+    headers = {}
+    if len(hits) == 1:
+        alert = hits[0][0]
+        disable_url = reverse(
+            "one_click_disable_alert", args=[alert.secret_key]
+        )
+        headers = {
+            "List-Unsubscribe-Post": "List-Unsubscribe=One-Click",
+            "List-Unsubscribe": f"<https://www.courtlistener.com{disable_url}>",
+        }
+    else:
+        params = {"keys": [hit[0].secret_key for hit in hits]}
+        query_string = urllib.parse.urlencode(params, doseq=True)
+        alert_list = reverse("disable_alert_list")
+        headers = {
+            "List-Unsubscribe": f"<https://www.courtlistener.com{alert_list}?{query_string}>",
+        }
```

Comment:
This is a quibble, but how about:

```suggestion
    headers = {}
    query_string = ""
    if len(hits) == 1:
        alert = hits[0][0]
        unsubscribe_path = reverse(
            "one_click_disable_alert", args=[alert.secret_key]
        )
        headers["List-Unsubscribe-Post"] = "List-Unsubscribe=One-Click",
    else:
        params = {"keys": [hit[0].secret_key for hit in hits]}
        query_string = urlencode(params, doseq=True)
        unsubscribe_path = reverse("disable_alert_list")
    headers["List-Unsubscribe"] = f"<https://www.courtlistener.com{unsubscribe_path}?{query_string}>"
```

Reply:
Your suggestion definitely looks better. Thank you, I'll go with that.